Makefile

```diff
@@ -148,4 +148,8 @@ tokengen:
 
 .PHONY: idemixgen
 txgen:
-	@go install github.com/IBM/idemix/tools/idemixgen
\ No newline at end of file
+	@go install github.com/IBM/idemix/tools/idemixgen
```

Comment:
I think you will need a specific version here. the problem is that the dependency sometimes is updated because of other dependencies during a go mod. then you have to reset it back to that specific version needed.

Reply:
This line stayed as-is and out of the scope of this PR